moped-database/migrations/1676056649901_dedupe_subcomponents_table/up.sql

```diff
@@ -0,0 +1,53 @@
+-- copy component <> subcomponent relationships from public.moped_subcomponents
+INSERT INTO public.moped_components_subcomponents (component_id, subcomponent_id)
+SELECT
+    component_id,
+    subcomponent_id
+FROM
+    public.moped_subcomponents ORDER BY subcomponent_id asc;
+
+-- update references to dupe subcomponents (except raised pavement marker)
+update public.moped_proj_components_subcomponents set subcomponent_id = 1 where subcomponent_id = 4;
+update public.moped_proj_components_subcomponents set subcomponent_id = 2 where subcomponent_id = 5;
+update public.moped_proj_components_subcomponents set subcomponent_id = 3 where subcomponent_id = 6;
+update public.moped_proj_components_subcomponents set subcomponent_id = 1 where subcomponent_id = 7;
+update public.moped_proj_components_subcomponents set subcomponent_id = 2 where subcomponent_id = 8;
+update public.moped_proj_components_subcomponents set subcomponent_id = 3 where subcomponent_id = 9;
```

Comment:
we do not have event triggers configured for this table. if we did, we would want to strongly consider disabling the trigger during these bulk updates.